qa/vagrant/build.gradle

```diff
@@ -64,7 +64,8 @@ String upgradeFromVersion
 
 String maybeTestsSeed = System.getProperty("tests.seed", null);
 if (maybeTestsSeed != null) {
-  seed = new BigInteger(maybeTestsSeed, 16).longValue()
+  def (masterSeed, testSeed) = maybeTestsSeed.tokenize(':')
+  seed = new BigInteger(masterSeed, masterSeed.size()).longValue()
```

Comment:
@brwe I pulled your PR and it fails for me:

```
13:42:07 ⏚ ⌂89% [jason:~/src/elastic/elasticsearch] brwe-fix-seed-for-vagrant ± gradle packagingTest -Pvagrant.boxes=sles-12 -Dtests.seed=ABC
:buildSrc:compileJava UP-TO-DATE
:buildSrc:compileGroovy UP-TO-DATE
:buildSrc:writeVersionProperties UP-TO-DATE
:buildSrc:processResources UP-TO-DATE
:buildSrc:classes UP-TO-DATE
:buildSrc:jar UP-TO-DATE
:buildSrc:assemble UP-TO-DATE
:buildSrc:compileTestJava UP-TO-DATE
:buildSrc:compileTestGroovy UP-TO-DATE
:buildSrc:processTestResources UP-TO-DATE
:buildSrc:testClasses UP-TO-DATE
:buildSrc:test UP-TO-DATE
:buildSrc:check UP-TO-DATE
:buildSrc:build UP-TO-DATE
=======================================
Elasticsearch Build Hamster says Hello!
=======================================
  Gradle Version        : 2.13
  OS Info               : Mac OS X 10.11.5 (x86_64)
  JDK Version           : Oracle Corporation 1.8.0_92 [Java HotSpot(TM) 64-Bit Server VM 25.92-b14]
  JAVA_HOME             : /Library/Java/JavaVirtualMachines/jdk1.8.0_92.jdk/Contents/Home

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/jason/src/elastic/elasticsearch/qa/vagrant/build.gradle' line: 70

* What went wrong:
A problem occurred evaluating project ':qa:vagrant'.
> For input string: "ABC"

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 13.272 secs
13:42:23 ⏚ ⌂78% [jason:~/src/elastic/elasticsearch] brwe-fix-seed-for-vagrant 1 ± 
```

Yet, if I make the input have length sixteen then it is fine. Maybe we are misunderstanding each other?